json4s/src/main/scala/org/http4s/json4s/Json4sInstances.scala

```diff
@@ -10,10 +10,10 @@ import scala.util.control.NonFatal
 import scalaz.{EitherT, \/}
 
 trait Json4sInstances[J] {
-  implicit lazy val json: EntityDecoder[JValue] = jawn.jawnDecoder(facade)
+  implicit lazy val jsonDecoder: EntityDecoder[JValue] = jawn.jawnDecoder(facade)
```

Comment:
Probably just my habit of not putting eager vals in traits.  I don't think this needs to be lazy.